src/ota_metadata/file_table/__init__.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2022 TIER IV, INC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from ._orm import FTDirORM, FTNonRegularORM, FTRegularORM, FTRegularORMPool
+from ._table import (
+    FileTableDirectories,
+    FileTableNonRegularFiles,
+    FileTableRegularFiles,
+)
+from ._types import FileEntryAttrs
+
+__all__ = [
+    "FTNonRegularORM",
+    "FTRegularORM",
+    "FTDirORM",
+    "FTRegularORMPool",
```

Comment:
How about to use `FileTable` instead of `FT`? In my opinion, it is more obvious.
https://mcusoft.wordpress.com/wp-content/uploads/2015/04/the-art-of-readable-code.pdf
![image](https://github.com/user-attachments/assets/0a91cb76-164d-47c4-a838-850232b8e62d)

Reply:
:smiley: Yes sure! I also think I make the naming toooo abbreviated, let's shorten it.

Reply:
Updated at [a5aac71](https://github.com/tier4/ota-client/pull/460/commits/a5aac713381e2c15482c858359c181cc9811cab0).